### PR TITLE
Create events_key_state_handler.py

### PR DESCRIPTION
**Pull request by Hibot12**
As intended in #239, this PR proposes creating another example of using KeyStateHandler.

### CODE REVIEW

File: examples/programming_guide/events_key_state_handler.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import pyglet
+
+window = pyglet.window.Window()
+
+key_handler = pyglet.window.key.KeyStateHandler()
+window.push_handlers(key_handler)
+
+
+def update(*args):
```
Comment (benmoran56) [inline examples/programming_guide/events_key_state_handler.py:52]: To to be clear what's being passed, could you change *args to "dt"? Even if it's unused, it shows what the update method is being passed.

File: examples/programming_guide/events_key_state_handler.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+"""
+"""
+
+from __future__ import print_function
+
+__docformat__ = 'restructuredtext'
+__version__ = '$Id$'
+
+import pyglet
```
Comment (benmoran56) [inline examples/programming_guide/events_key_state_handler.py:44]: Could you remove everything above the "import pyglet" line, and below the license section? The future import and docstring declarations are not needed anymore.   I know that they're still in the other examples, but that will get cleaned out eventually.

**Comment by benmoran56** [comment]
Thank you!  I made a few comments, but other than that it looks good. If you make new commits on your end, it will automatically update the PR.

**Comment by DoAltPlusF4** [comment]
In the docstring I would recommend making it clear that the KeyStateHandler is more of a polling approach to getting user input, and that both methods have different use cases.

File: examples/programming_guide/events_key_state_handler.py (hunk 1 above)
Comment (Hibot12) [reply examples/programming_guide/events_key_state_handler.py:52]: Yep. Doing it right now...

File: examples/programming_guide/events_key_state_handler.py (hunk 2 above)
Comment (Hibot12) [reply examples/programming_guide/events_key_state_handler.py:36]: Done.

**Comment by Hibot12** [comment]
Sounds great for me, @DoAltPlusF4 .

**Comment by benmoran56** [comment]
Thanks for the changes. Merged!

**Pull request merged**